### ISSUE TITLE
Feature Request: Add settings for execute on save

### ISSUE THREAD

**Issue by JWess** [gaoshan0621/vscode-csharp-format-usings#8]
I would use this extension if it provided a way to sort/remove usings on save. Visual Studio 2022 can do this via "Run Code Cleanup profile on save", but I want to use VSCode.

**Comment by kuiperzone** [comment]
Indeed, I second this and miss the clean up option for VS. I forever removing redundant usings.

Any of these would be great:

1. Remove on save (think this is easiest and sufficient), or:

2. A right-click option to remove all in open windows

3. Or an option to clean up project/solution

**Comment by kuiperzone** [comment]
BTW - good extension and a life-saver . Thank you!
